app/scripts/models/auth_brokers/base.js

```diff
@@ -31,7 +31,13 @@ define([
       var self = this;
       return p()
         .then(function () {
-          self._isForceAuth = self._isForceAuthUrl();
+          self._isForceAuth = self._isForceAuthUrl() ||
+                              self._isSignInWithForceEmail();
+
+          // The `/force_auth` screen expects `email`, not `forceEmail`
+          if (self._isSignInWithForceEmail()) {
+            self.relier.set('email', self.relier.get('forceEmail'));
```

Comment:
I'm not sure how I feel about this API decision of trigger this feature using the `forceEmail` param. A couple concerns:

* I'm not sure what the semantics are to be if the relier specifies both an `email` and a `force_email`.  
* Lines like `self.relier.set('email', self.relier.get('forceEmail'))` seem hacky to me. It suggests we should just be re-using the existing `email` param and add another param (`force_auth`?) to signal that the relier doesn't want the user to be able to change it.